### ISSUE TITLE
 error - Undefined property: item::$tags

### ISSUE THREAD

**Issue by chrispowell100** [nanostudio-org/nanogallery2#262]
nanogallery2: Could not retrieve nanoPhotosProvider2 data. Error: error - Undefined property: item::$tags
(nano_photos_provider2.json.class.php/1122) [nanogallery2]

Msg above is received when trying to display album ('Elsenham Old') contents. I have removed images randomly from the folder and error persists right down to last image. Have deleted and recreated folder and loaded single, random images into it and error immediately re-occurs. If I copy same image(s) into another folder they show OK - no error msgs. If I copy 'good' images into this folder, error re-occurs.
I have looked at the meta tags for the images in that folder and they appear no different to images in any other folder.
![NG problem - Copy](https://user-images.githubusercontent.com/2922882/83874504-7bc38700-a72d-11ea-86e7-9fa300bc6ee1.JPG)

**To Reproduce**
1. Go to 'http://www.elsenham-history.co.uk/nanogallery2/gallery_N.php'
2. Click on 'Elsenham Old' to get error msg. All other albums work OK

**Expected behavior**
  
**CODEPEN**
Please provide a Codepen to reproduce your issue.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 
 - Browser - Chrome/Firefox/Edge

**Smartphone (please complete the following information):**
 - Device: iPhone6
 - OS:  iOS13.4
 - Browser Safari/Chrome

**Additional context**

**Comment by chrispowell100** [comment]
Some more info. I should have added in my original entry that each time I added/deleted an image to the 'Elsenham Old' folder, I deleted the  _thumbnails folder in both 'Elsenham Old' and in the 'nano_photos_content' parent folder.

I have also tried running in development mode and got the stream of PHP messages shown in attached 'NG problem - dev mode' image.
![NG problem - dev mode](https://user-images.githubusercontent.com/2922882/83940439-ee8b3b80-a7db-11ea-9a8b-75cf874bf758.JPG)
My level of knowledge of PHP is not good enough to make much sense of this.

When in production mode the contents of the 'Elsenham Old _thumbnails' folder are as in attached 
'NG problem - thumbnails prod mode.JPG'
![NG problem - thumbnails prod mode](https://user-images.githubusercontent.com/2922882/83940673-dddbc500-a7dd-11ea-9bab-739a93519ed3.JPG)

When in dev mode 'Elsenham Old _thumbnails' folder contents are different as in attached 'NG problem - thumbnails dev mode.JPG'
![NG problem - thumbnails dev mode](https://user-images.githubusercontent.com/2922882/83940688-f815a300-a7dd-11ea-9dec-db7a94a85aaa.JPG)

But in neither case does it open the folder.
